zilliqa/src/api/types/eth.rs

```diff
@@ -101,13 +101,14 @@ pub struct Block {
     pub quorum_certificate: QuorumCertificate,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub aggregate_quorum_certificate: Option<AggregateQc>,
+    pub version: u64,
```

Comment:
Why did we add this field? It isn't documented in any of the `eth_getBlockByNumber` API docs I could find.

Reply:
Just for ZQ1 backwards compatibility

Reply:
Fixed here: https://github.com/Zilliqa/zq2/pull/1789